src/canvas/canvas.class.ts

```diff
@@ -833,10 +836,7 @@ export class Canvas<
     let pointer = this.getPointer(e);
     if (target.group) {
       // transform pointer to target's containing coordinate plane
-      // should we use send point to plane?
-      pointer = pointer.transform(
-        invertTransform(target.group.calcTransformMatrix())
-      );
+      pointer = sendPointToPlane(pointer, target.group.calcTransformMatrix());
```

Comment:
here

Reply:
should be `sendPointToPlane(pointer, undefined, target.group.calcTransformMatrix());`